### ISSUE TITLE
Firefox extension nightly builds fail to be signed by Mozilla

### ISSUE THREAD

**Issue by mossroy** [kiwix/kiwix-js#314]
When Travis runs the script that asks Mozilla to sign the extension, there is an error : 
"/version" should match format "versionString"
Error: Your JSON file could not be parsed.

I suppose Mozilla changed its policy on this version string : I have to adapt the script for that
https://developer.mozilla.org/en-US/docs/Mozilla/Toolkit_version_format

**Comment by Jaifroid** [comment]
While adapting the scripts to build the bundle version of the app, I discovered that Firefox nightlies have not been signed by the API, probably for many months or even years.

While I was able to get web-ext to initiate the signing process, but the API returns "Oops, you tried to do something you are not allowed to do" (or something similar).

I checked back as far as I could, and the last nightly run I could access was from April. It shows that nightlies were not being successfully run then either, so this is nothing to do with the bundling code.

I have commented out the code, but have left a notice to DEVs, in case someone wants to come back to this in the future. I'm not sure having signed nightlies is that useful anyway.
